src/pytorch_lightning/trainer/trainer.py

```diff
@@ -2207,11 +2206,11 @@ def model(self, model: torch.nn.Module) -> None:
 
     @property
     def log_dir(self) -> Optional[str]:
-        if len(self.loggers) == 1:
-            if isinstance(self.logger, TensorBoardLogger):
-                dirpath = self.logger.log_dir
+        if len(self.loggers) > 0:
+            if isinstance(self.loggers[0], TensorBoardLogger):
```

Comment:
can we add this info inside profiler docs?

Reply:
I find the profilers arguments doc has mentioned where to store profiles If `dirpath` is `None` but `filename` is present: https://github.com/Lightning-AI/lightning/blob/master/src/pytorch_lightning/profilers/simple.py#L46-L48

I'm not sure where should we add. At https://github.com/Lightning-AI/lightning/blob/master/docs/source-pytorch/tuning/profiler_basic.rst?plain=1#L100?

Reply:
yes, seems reasonable